client/src/containers/explore/section/index.tsx

```diff
@@ -72,59 +46,64 @@ const Section: React.FC<PropsWithChildren<SectionProps>> = ({
-                    fill="#3048B5"
-                  />
-                </svg>
-              </div>
-            </Button>
-          </PopoverTrigger>
-          <PopoverContent align="end" id={POPOVER_ID}>
-            {menuItems.map((s) => (
-              <Button
-                key={`menu-item-${s.slug}`}
-                variant="clean"
-                className="block w-full p-4 text-left text-xs hover:bg-primary-foreground"
-                onClick={() => {
-                  handleLinkClick(s.slug);
-                }}
-              >
-                {s.name}
+      ) : (
+        <header className="flex justify-between space-y-4 rounded-2xl bg-secondary p-6">
+          <div>
+            <h2 className="text-xl font-semibold">{name}</h2>
+            <p className="text-muted-foreground">{description}</p>
+          </div>
+          <Popover open={popoverOpen} onOpenChange={setPopOverOpen}>
+            <PopoverTrigger asChild id="section-nav">
+              <Button type="button" variant="default" className="p-0">
+                <div className="flex items-center justify-center">
+                  <svg
```

Comment:
Perhaps we can move the svg to a `components/svg` folder and import it from there.